### DISCUSSION TITLE
charging to start on dynamic energy pricing.

### DISCUSSION THREAD

**Discussion by HansNetcom** [serkri/SmartEVSE-3#139] [Ideas]
Would it be possible to pickup the daily energie pricing from a dynamic energy contract an start a charging session based upon the lowest price level?

**Comment by tomcoonen** [comment]
You can do this in your home automation software? Seems like a more logical place to me.

**Comment by dingo35** [comment]
Currently you'd better do this in Home Assistant, that is more flexible. Once there is an etablished, crystallized user interface we can implement that in the firmware...

**Comment by HansNetcom** [comment]
Ok,I will give it a try
